src/UnityExtension/Assets/Editor/GitHub.Unity/UI/SettingsView.cs

```diff
@@ -240,12 +243,11 @@ private void ResetToDefaults()
             gitName = Repository != null ? Repository.User.Name : String.Empty;
             gitEmail = Repository != null ? Repository.User.Email : String.Empty;
             repositoryRemoteName = DefaultRepositoryRemoteName;
-            repositoryRemoteUrl = string.Empty;
+            newRepositoryRemoteUrl = repositoryRemoteUrl = string.Empty;
```

Comment:
`newRepositoryRemoteUrl` and `repositoryRemoteUrl` are both used..

https://github.com/github-for-unity/Unity/blob/4a208bd88f1396e135a89088dd1f25d69b96b653/src/UnityExtension/Assets/Editor/GitHub.Unity/UI/SettingsView.cs#L344-L345